### PR TITLE
add new flag scan external jar package

### PR DESCRIPTION
**Pull request by cyemars**
### Describe what this PR does / why we need it
解决Java工程无法调用外部jar 包

### Does this pull request fix one issue?
添加flag 指定外部jar 包扫描路径

### Describe how you did it
在agent挂载的时候传入jar包路径  扫描二方包

### CODE REVIEW

**Comment by cyemars** [comment]
肖老师  我第一次接触go 请你指导一下

**Comment by tiny-x** [comment]
> ### Describe what this PR does / why we need it
> 解决Java工程无法调用外部jar 包
> 
> ### Does this pull request fix one issue?
> 添加flag 指定外部jar 包扫描路径
> 
> ### Describe how you did it
> 在agent挂载的时候传入jar包路径 扫描二方包

````
./blade -h
````

panic: "jp" shorthand is more than one ASCII character

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc000279400, 0xc000583360)
	/Users/yefei/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:864 +0x749
github.com/spf13/pflag.(*FlagSet).VarPF(0xc000279400, 0x539acc0, 0xc000193cd0, 0x51a1a92, 0x7, 0x519f61a, 0x2, 0x51b0af2, 0x16, 0xc0005832c0)
	/Users/yefei/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:831 +0x10b
github.com/spf13/pflag.(*FlagSet).VarP(...)
	/Users/yefei/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:837
github.com/spf13/pflag.(*FlagSet).StringVarP(0xc000279400, 0xc000193cd0, 0x51a1a92, 0x7, 0x519f61a, 0x2, 0x0, 0x0, 0x51b0af2, 0x16)
	/Users/yefei/go/pkg/mod/github.com/spf13/pflag@v1.0.5/string.go:42 +0xad
github.com/chaosblade-io/chaosblade/cli/cmd.(*PrepareJvmCommand).Init(0xc000193c80)
	/temp/chaosblade/cli/cmd/prepare_jvm.go:56 +0x1d2
github.com/chaosblade-io/chaosblade/cli/cmd.(*baseCommand).AddCommand(0xc00019e4c8, 0x5398a00, 0xc000193c80)
	/temp/chaosblade/cli/cmd/command.go:136 +0x35
github.com/chaosblade-io/chaosblade/cli/cmd.CmdInit(0xc00077bf48)
	/temp/chaosblade/cli/cmd/cmd.go:33 +0x102
main.main()
	/temp/chaosblade/cli/main.go:27 +0x26

File: cli/cmd/prepare_jvm.go [hunk 1]
```diff
@@ -50,7 +51,9 @@ func (pc *PrepareJvmCommand) Init() {
 		},
 		Example: pc.prepareExample(),
 	}
+	//chaosbaldeJarPath  the extrenal jar path
 	pc.command.Flags().StringVarP(&pc.javaHome, "javaHome", "j", "", "the java jdk home path")
+	pc.command.Flags().StringVarP(&pc.chaosbladeJarPath, "jarPath", "jp", "", "the java jdk home path")
```
Comment (tiny-x) [inline cli/cmd/prepare_jvm.go:56]: "jp" shorthand is more than one ASCII character

**Comment by tiny-x** [comment]
@cyemars This scheme seems not to work. -Xbootclasspath is only started for the JVM that loads the agent, not the JVM that affects the application

**Comment by tiny-x** [comment]
https://github.com/chaosblade-io/chaosblade-exec-jvm/pull/138

**Pull request closed without merging**